compiler/x/codegen/OMRCodeGenerator.cpp

```diff
@@ -995,9 +995,23 @@ OMR::X86::CodeGenerator::getSupportsOpCodeForAutoSIMD(TR::ILOpCode opcode, TR::D
     * The cases that return false are placeholders that should be updated as support for more vector evaluators is added.
     */
    // implemented vector opcodes
+
+   if (opcode.isVectorOpCode() && opcode.getVectorOperation() == OMR::vadd)
+      {
+      TR::DataType ot = opcode.getVectorResultDataType();
```

Comment:
Shouldn't length and opcode.getVectorResultDataType() be equal? Also, for the old vector opcodes, the vector length is assumed to be 128, so do we really need that argument?

Reply:
Good points. I was thinking of removing that argument when we remove all old style opcodes. But maybe you are right, we can remove it now.

Reply:
done